src/states/node.rs

```diff
@@ -3418,6 +3418,13 @@ impl Node {
         self.clients_rate_limiter.get_clients_usage()
     }
 
+    pub fn get_banned_client_ips(&self) -> BTreeSet<IpAddr> {
+        self.banned_client_ips
+            .peek_iter()
+            .map(|(ip, _)| *ip)
+            .collect()
```

Comment:
Well, maybe `peek_keys()` and `keys()`, but can't say I'm too bothered either way.